wandb/cli/cli.py

```diff
@@ -886,7 +938,8 @@ def _parse_settings(settings):
                 "queue": queue,
                 "run_spec": json.dumps(
                     construct_launch_spec(
-                        os.getcwd(),  # uri,
+                        None, # uri
+                        job,  # job
```

Comment:
Why are we passing in the job as the job for the launch scheduler and also for the entry_point shouldn't it only need one?

Reply:
The scheduler needs to know what the job is to add runs to the launch queue. What I think might not need a job is the launch spec, which is for the scheduler itself (which shouldn't have a job? or at least be a different job?)